From 4028320ba2e8c181292e02171d83ebf6ce3aee5a Mon Sep 17 00:00:00 2001
From: Felipe Contreras <felipe.contreras@gmail.com>
Date: Thu, 29 Dec 2011 03:28:27 +0200
Subject: [PATCH] gtk: conf: set correct download dir default

Otherwise people might get:
ERROR:open-dialog.c:285:gtr_torrent_options_dialog_new: code should not be reached

It was fixed at some point in ticket #1873, but somebody broket it
again.

Signed-off-by: Felipe Contreras <felipe.contreras@gmail.com>
---
 gtk/conf.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/gtk/conf.c b/gtk/conf.c
index 623b839..6ad6ca2 100644
--- a/gtk/conf.c
+++ b/gtk/conf.c
@@ -108,7 +108,6 @@ tr_prefs_init_defaults( tr_benc * d )
     tr_bencDictAddInt( d, PREF_KEY_MAIN_WINDOW_X, 50 );
     tr_bencDictAddInt( d, PREF_KEY_MAIN_WINDOW_Y, 50 );
 
-    str = g_get_user_special_dir( G_USER_DIRECTORY_DOWNLOAD );
     tr_bencDictAddStr( d, TR_PREFS_KEY_DOWNLOAD_DIR, str );
 
     tr_bencDictAddStr( d, PREF_KEY_SORT_MODE, "sort-by-name" );
-- 
1.7.8.1

